Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -284,6 +287,75 @@ enum PackageController {
         return response
     }
 
+    static func siteMap(req: Request) async throws -> Response {
```

Comment:
This method is too long. May need a refactor before merging, but let's discuss.

Reply:
TBH I think it's fine but it looks like there's a nested `func linkableEntityUrls ` that could easily be pulled out into a sibling `static func`?